libs/elements/layout/images/src/lib/components/image-container/image-container.component.ts

```diff
@@ -0,0 +1,65 @@
+import { Component, Input } from '@angular/core';
+
+import { ImageConfig } from '@elewa-website/models/schema/ui/images';
+
+@Component({
+  selector: 'elewa-website-image-container',
+  templateUrl: './image-container.component.html',
+  styleUrls: ['./image-container.component.scss'],
+})
+export class ImageContainerComponent {
+  private _imageConfig!: ImageConfig;
+
+  /**
+   * Property to store the class to be used string.
+   */
+  imageClassName = '';
+  
+
+  /**
+   * Setter for the `imageConfig` input property.
+   * This method is automatically called when the `imageConfig` input is set externally.
+   * It updates the private `_imageConfig` property and formats the price using commas.
+   * 
+   * @param {PriceItem} value - The new value for the `imageConfig` input.
+   */
+  @Input()
+  set imageConfig(value: ImageConfig) {
+    this._imageConfig = value;
+    this.imageClassName = this.getClassName(value.visualisation);
+  }
+
```

Comment:
There's no need to have this as a setter since the value will never change after it's initially passed in. We can just pass it in as input.